modules/twinklexfd.js

```diff
@@ -1443,7 +1443,8 @@ Twinkle.xfd.callbacks = {
 			var params = pageobj.getCallbackParameters();
 			var statelem = pageobj.getStatusElement();
 
-			var newtext = text.replace('---- and enter on a new line, directly below -->', '---- and enter on a new line, directly below -->\n' + Twinkle.xfd.callbacks.getDiscussionWikitext('rm', params));
+			var hiddenComment = '---- and enter on a new line, directly below this line; do not add spare lines between entries, for accessibility reasons -->';
```

Comment:
```suggestion
			var hiddenComment = '---- and enter on a new line, directly below; do not add spare lines between entries, for accessibility reasons -->';
```

https://en.wikipedia.org/wiki/Wikipedia_talk:Twinkle#%22Adding_entry_at_WP:RM/TR:_failed_to_find_target_spot_for_the_entry%22